plugins/PRAplugin/src/MCSSolver.py

```diff
@@ -109,14 +109,13 @@ def createNewInput(self, container, inputs, samplerType, **Kwargs):
       # E.g., for order=2: [ (['A', 'B', 'C'], ['D', 'C']),
       #                      (['A', 'B', 'C'], ['A', 'E']),
       #                      (['D', 'C'], ['A', 'E']) ]
-      self.topEventTerms[order] = []
-      for term in terms:
-        basicEventCombined = set(itertools.chain(*term))
-        # E.g. if                 term=(['A', 'B', 'C'], ['D', 'C'])
-        #      then basicEventCombined={'A', 'B', 'D', 'C'}
-        if basicEventCombined not in self.topEventTerms[order]:
-          self.topEventTerms[order].append(list(basicEventCombined))
-    return Kwargs
+
+      basicEventCombined = list(set(itertools.chain.from_iterable(term)) for term in terms)
+      self.topEventTerms[order]=basicEventCombined
+
+    for term in self.topEventTerms:
+      print(term,self.topEventTerms[term])
```

Comment:
remove the for loop and print, this is for debug, right?

Reply:
yup, removed